### DISCUSSION TITLE
开始菜单风格切换为 Windows 10 时，应用列表无法显示任何UWP应用或者任何非win32应用（包括系统自带应用在内）

### DISCUSSION THREAD

**Discussion by mushMario** [zetaloop/ExplorerPatcher#17] [Q&A] [answered]
大概是有一次试图调整开始菜单样式的时候，只是在开始菜单风格来回切换
换回来就发现所有 UWP 应用在开始菜单上的应用列表里不见了，在开始菜单上其对应的磁铁也消失了
但所有应用实际上还在，切回 Windows 11 开始菜单也依然能够正常看见这些 UWP 应用
请问我应该如何做，才能够正常地在 Windows 10 样式开始菜单里显示完整的应用列表？

**Comment by zetaloop** [comment]
我现在在用 win11 25276.1000 (dev)，已经没办法换成win10开始菜单，选win10直接黑屏。[原作](https://github.com/valinet/ExplorerPatcher)已经两个月没更新了，可能是微软对win11的一些新的改动所致，需要原作者继续适配，但是我不会做。
ExplorerPatcher的修改稳定性不高，如果您迫切需要这样的功能，可以尝试旧版、官方版，或者其他类似软件，感谢关注。

**Reply by mushMario** [reply]
感谢回复
~~微软天天整了些什么烂活~~

**Comment by Alcor-Lu** [comment, marked as answer]
同样遇到了，在虚拟机上复现后发现是22H2的1月更新导致的，此前版本没有问题，目前我的临时解决办法是：先卸载，用22H2的12月镜像回退系统版本，然后重新安装，暂停系统更新。如果安装了1月更新，如果不重新改回win11开始菜单也不会出现问题。

**Reply by ghost** [reply]
也就是说 在旧版Win11改了Win10开始菜单之后 再更新的话 就没事了 是吗？

**Reply by Alcor-Lu** [reply]
不完全是，在旧版本改win10菜单再更新只能使已经存在的UWP应用不会消失，此后新安装的UWP应用不会出现在开始菜单中

**Reply by Alcor-Lu** [reply]
另外知乎上有人给了一个临时解决办法，https://zhuanlan.zhihu.com/p/599158074

**Reply by ghost** [reply]
好的 谢谢

**Comment by ghost** [comment]
我也有这个问题。可能是Win11底层问题。等待更新吧。

**Comment by Zeyun-Deng** [comment]
我也遇到这个问题了。valinet大神的discussion也关闭了，只能在这里回复了🙏无奈切换回win11的开始菜单

**Comment by KIITTID** [comment]
我也是这个问题，最新预发行版也不行，有没有法子啊

**Comment by zetaloop** [comment]
已解决，请使用最新的 [修复包v3.0](https://github.com/zetaloop/ExplorerPatcher/files/12137341/v3.0.zip)

**Reply by Alcor-Lu** [reply]
非常感谢，实测可用，但还有一些小问题：不能使用右键固定到开始菜单的方式固定磁贴，只能通过鼠标拖动的方式固定。固定的磁贴不能取消固定，但是可以通过把要取消固定的磁贴移动到一个新组，然后删除组的方式来取消固定。

**Reply by zetaloop** [reply]
@Alcor-Lu 对的。https://github.com/zetaloop/ExplorerPatcher/issues/6#issuecomment-1646692951
